SparkPost/2web2ui
# FE-1096 - Incorporate UX feedback for the templates feature

[FE-1096](https://jira.int.messagesystems.com/browse/FE-1096)

### What Changed
- Incorporated updates to the templates feature according to UX requirements

### How To Test
- Log in to the usual test account and go to `/templates`:
  - [ ] Verify the "Recent Activity" heading is smaller, the same size as the one on the "Alerts" page
  - [ ] Verify that the divider within the `<Panel/>` was removed from the top of the "Recent Activity" panels
- Go to `/templates/edit/fe-562/draft/content`:
  - [ ] Verify that a paper airplane icon renders within the "Send a Test" button
  - [ ] Verify the "Template Settings" heading no longer renders within the "Template Settings" form
  - [ ] Verify there are no section dividers within the "Template Settings" form
- [ ] Verify no bugs were introduced because of the changes!

### To Do
